src/metabase/models/setting.clj

```diff
@@ -370,11 +370,14 @@
 
 (defn- user-facing-info [setting]
   (let [k (:name setting)
-        v (get k)]
+        v (get k)
+        set-by-env (not (nil? (env-var-value setting)))]
```

Comment:
also since `env-var-value` is guaranteed to return either a non-empty string or `nil` you can just do

`(boolean (env-var-value setting))` which is the Clojure equivalent of the C `!!` idiom. Simpler than calling `(not (nil? ...))`